### PR TITLE
Issue 5 view list of logged workouts

### PR DESCRIPTION
**Pull request by rachelbeacham**
Screen Shot 2020-12-31 at 5 48 42 PM
Screen Shot 2020-12-31 at 5 50 07 PM

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -85,3 +102,12 @@ app.listen(process.env.PORT, () => {
...
 });
+
+/*
+select *
+    from "workouts"
+    join "exerciseSets" using ("workoutId")
+    join "exercises" using ("exerciseId")
+    join "sets" using ("setId")
+   where "userId" = $1
+   */
```
Comment (thebearingedge) [inline server/index.js:113]: # ♻️

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, great job on the code. Your workouts could use some space between the content and the bottom edge though.

File: server/index.js (hunk 1 above)
Comment (rachelbeacham) [reply server/index.js:113]: 😂  I save stuff all the way at the bottom in comments then forget about it, whoops!!

**Review by thebearingedge — APPROVED** [review]

**Comment by rachelbeacham** [comment]
Screen Shot 2020-12-31 at 6 04 13 PM

**Pull request merged**